lib/src/main/java/com/squareup/cycler/RecyclerMutations.kt

```diff
@@ -257,10 +342,76 @@ private class MutationExtension<T : Any>(val spec: MutationExtensionSpec<T>) : E
       lastActionState = actionState
     }
 
+    private fun removeSwipeState() {
+      swipeState = null
+    }
+
+    private fun createSwipeStateFor(dataItem: T, itemView: View) {
+      // TODO: particular underView specs could be added to
+      // main-recycler's row specs, to be used for those items only
+      // and keep things grouped by type of item (in the main recycler).
+      val underViewRowSpec = spec.swipeUnderViewSpecs.firstOrNull { it.matches(dataItem) }
+      underViewRowSpec?.createViewHolder(
+          CreatorContext(recycler), // recycler is not exactly correct, but well enoguh for now
```

Comment:
Nit: What value does this comment add? It doesn't explain _why_ it's not correct, nor why it's good enough for now. Should answer at least one, preferably both, of those questions or not exist at all.